proto/viam/app/packages/v1/packages.proto

```diff
@@ -103,6 +105,7 @@ message GetPackageRequest {
 
 message GetPackageResponse {
   Package package = 1;
+  string version = 2;
```

Comment:
Actually this may not be needed. I didn't notice but PackageInfo has `string version = 3; `. I think that would suffice.

Reply:
You're right, removed.